src/Space.mockClient.test.ts

```diff
@@ -93,26 +93,26 @@ describe('Space (mockClient)', () => {
       space.on('membersUpdate', callbackSpy);
 
       space.dispatchEvent(createPresenceEvent('enter'));
-      expect(callbackSpy).toHaveBeenNthCalledWith(1, [
+      expect(callbackSpy).toHaveBeenNthCalledWith<SpaceMember[][]>(1, [
```

Comment:
Is the type `SpaceMember[][]` because the call happens multiple times and vitest puts it into an array?

Reply:
the function can accept any number of arguments, so the array part is just an array of the arguments